make-containerized-build.groovy

```diff
@@ -27,12 +23,18 @@ node('cico-workspace') {
 
 	try {
 		stage('prepare bare-metal machine') {
-			sh 'scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ./prepare.sh root@${CICO_NODE}:'
-			sh 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} ./prepare.sh --workdir=/opt/build --gitrepo=${GIT_REPO} --branch=${GIT_BRANCH}'
-		}
-
+			if ("${ghprbPullId}".length() == 0){
```

Comment:
Can you change this if, so that it only sets the `ref` in case `ghprbPullId` is not set?